accesspoc/dips/tasks.py

```diff
@@ -0,0 +1,33 @@
+from celery import shared_task
+from .parsemets import METS
+
+import os
+import re
+import shutil
+import tempfile
+import zipfile
+
+
+@shared_task
+def extract_and_parse_mets(dip_id, zip_path):
```

Comment:
Can we add a test?
http://docs.celeryproject.org/en/latest/userguide/testing.html